libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -602,6 +604,7 @@ const RC_Channel::LookupTable RC_Channel::lookuptable[] = {
     { AUX_FUNC::EKF_POS_SOURCE,"EKFPosSource"},
     { AUX_FUNC::CAM_MODE_TOGGLE,"CamModeToggle"},
     { AUX_FUNC::GENERATOR,"Generator"},
+    { AUX_FUNC::BATTERY_MPPT_ENABLE,"Battery MPPT Enable"},
```

Comment:
Can we define this out with something? String costs flash for everyone.

Reply:
At some point I'll need to pull out MPPT into it's own library and/or battery type and that will be easier to enable/disable